openhcl/underhill_attestation/src/igvm_attest/mod.rs

```diff
@@ -24,26 +24,16 @@ base64_serde_type!(Base64Url, base64::engine::general_purpose::URL_SAFE_NO_PAD);
 
 #[allow(missing_docs)] // self-explanatory fields
 #[derive(Debug, Error)]
```

Comment:
The intention for this pub enum was to defining the list of errors to be exposed outside of this crate. I think it’s still better to keep this instead of  creating another pub enum for Ak cert.

Reply:
I was trying to scope it down given that they are only used by the AkCert file.